app/helpers/orders_helper.rb

```diff
@@ -23,6 +23,15 @@ def options_for_suppliers_to_select
     options_for_select(options)
   end
 
+  def format_amount(amount, order_article, strip_insignificant_zeros: false)
+    strip_insignificant_zeros = true unless order_article.article_version.supplier_order_unit_is_si_convertible
+    number_with_precision(amount, precision: 3, strip_insignificant_zeros: strip_insignificant_zeros)
+  end
+
+  def format_units_to_order(order_article, strip_insignificant_zeros: false)
+    format_amount(order_article.units_to_order, order_article, strip_insignificant_zeros: strip_insignificant_zeros)
+  end
```

Comment:
Both those new methods might be used in other cases later on and don't really require an `order_article` to work - they should rather take `article_version` as parameter. Subsequently they should be moved to the `ArticlesHelper`, which currently is the designated helper for both `Article` and `ArticleVersion`.